test/framework/src/main/java/org/opensearch/snapshots/AbstractSnapshotIntegTestCase.java

```diff
@@ -121,6 +127,8 @@ protected Settings nodeSettings(int nodeOrdinal) {
             // Rebalancing is causing some checks after restore to randomly fail
             // due to https://github.com/elastic/elasticsearch/issues/9421
             .put(EnableAllocationDecider.CLUSTER_ROUTING_REBALANCE_ENABLE_SETTING.getKey(), EnableAllocationDecider.Rebalance.NONE)
+            .put(FeatureFlags.SEGMENT_REPLICATION_EXPERIMENTAL, "true")
+            .put(FeatureFlags.REMOTE_STORE, "true")
```

Comment:
pls remove . We can't change existing node settings . Lets override for remote store specific test cases.